lib/modules/onboarding/onboarding.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace Podlove;
+
+namespace Podlove\Modules\Onboarding;
+
+use Podlove\Modules\Onboarding\Settings\OnboardingPage;
+
+class Onboarding extends \Podlove\Modules\Base
+{
+    protected $module_name = 'Onboarding';
+    protected $module_description = 'Handling the onboarding to the Podlove Publisher';
+    protected $module_group = 'Getting Started';
```

Comment:
Vielleicht zu "system"? Da ist z.B. auch das alte Migrationsmodul.

```suggestion
    protected $module_group = 'system';
```

Reply:
Done